lib/SPIRV/LLVMBitreverse.h

```diff
@@ -0,0 +1,148 @@
+//===- LLVMBitreverse.h - implementation of llvm.bitreverse -===//
+//
+//                     The LLVM/SPIRV Translator
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+// Copyright (c) 2024 Intel Corporation. All rights reserved.
```

Comment:
Please use the following license: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/80#issuecomment-476781085

Reply:
License has been udpated.